scripts/sanitycheck

```diff
@@ -476,13 +476,14 @@ class SizeCalculator:
     alloc_sections = ["bss", "noinit", "app_bss", "app_noinit"]
     rw_sections = ["datas", "initlevel", "_k_task_list", "_k_event_list",
                    "_k_memory_pool", "exceptions", "initshell",
-		   "_static_thread_area", "_k_timer_area", "_k_work_area",
-		   "_k_mem_slab_area", "_k_mem_pool_area",
-		   "_k_sem_area", "_k_mutex_area", "_k_alert_area",
-		   "_k_fifo_area", "_k_lifo_area", "_k_stack_area",
-		   "_k_msgq_area", "_k_mbox_area", "_k_pipe_area",
+                   "_static_thread_area", "_k_timer_area", "_k_work_area",
+                   "_k_mem_slab_area", "_k_mem_pool_area",
+                   "_k_sem_area", "_k_mutex_area", "_k_alert_area",
+                   "_k_fifo_area", "_k_lifo_area", "_k_stack_area",
+                   "_k_msgq_area", "_k_mbox_area", "_k_pipe_area",
                    "net_if", "net_if_event", "net_stack", "net_l2_data",
```

Comment:
Unrelated whitespace muckery snuck into the patch.

Reply:
doesn't the checkpatch script catch that?

Reply:
@agross-linaro catch what, exactly?
at any rate, the whitespace changes are in their own patch in the series.